snap/info.go

```diff
@@ -636,7 +639,8 @@ type AppInfo struct {
 	// TODO: this should go away once we have more plumbing and can change
 	// things vs refactor
 	// https://github.com/snapcore/snapd/pull/794#discussion_r58688496
-	BusName string
+	BusName  string
+	CommonId string
```

Comment:
I guess that as long as this common-id (at app-level) is serialised and available to clients then I'm happy.

Reply:
CommonID also I think it should go in the grouping with Name